lldb/include/lldb/Core/PluginManager.h

```diff
@@ -486,6 +544,12 @@ class PluginManager {
   static InstrumentationRuntimeCreateInstance
   GetInstrumentationRuntimeCreateCallbackAtIndex(uint32_t idx);
 
+  static std::vector<RegisteredPluginInfo>
+  GetInstrumentationRuntimePluginInfo();
```

Comment:
Did the system runtime stuff get removed from this patch?

Reply:
The SystemRuntime functions were added in an earlier PR that has already merged (#133794). That's why they are not showing up as changes in this PR.